src/shm_sub.c

```diff
@@ -41,6 +41,16 @@
 #include "sysrepo.h"
 #include "utils/nacm.h"
 
+struct sr_shmsub_oper_sub_s {
```

Comment:
Doxygen.

Reply:
Done

Reply:
Look into `shm_types.h` for examples of how to document a structure, it differs from a function.

Reply:
Fixed